config/config.go

```diff
@@ -89,6 +91,10 @@ func (cfg *LifecycleManagerConfig) ParseFlags() string {
 	kingpin.Flag("dry-run", "Don't make any changes, just print.").BoolVar(&cfg.DryRun)
 	kingpin.Flag("listen", "Address to listen at, e.g. :9090 or 0.0.0.0:9090").Default(defaultListener).StringVar(&cfg.Listen)
 	kingpin.Flag("workdir", "Path to working directory used for storing channel configurations.").Default(defaultWorkdir).StringVar(&cfg.Workdir)
+	kingpin.Flag(
+		"provider",
+		"Cloud provider. Defaults to single provider \"zalando-aws\".",
+	).Default(defaultProvider).EnumsVar(&cfg.Providers, "zalando-aws")
```

Comment:
Maybe use `ZalandoAWSProvider` const here?

Reply:
I tried at first, but got an annoying "import cycle not allowed" :slightly_frowning_face: , because `provisioner` package imports `config`.  

https://github.com/zalando-incubator/cluster-lifecycle-manager/blob/de834d61afe158c963eecc128a581ba2bb320fb0/provisioner/clusterpy.go#L21